src/main/java/seedu/address/model/person/Person.java

```diff
@@ -16,52 +11,40 @@
 public class Person {
 
     // Identity fields
+    private final Type type;
+    private final Nric nric;
     private final Name name;
     private final Phone phone;
-    private final Email email;
-
-    // Data fields
-    private final Address address;
-    private final Set<Tag> tags = new HashSet<>();
 
     /**
      * Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
-        requireAllNonNull(name, phone, email, address, tags);
+    public Person(Type type, Nric nric, Name name, Phone phone) {
+        requireAllNonNull(type, nric, name, phone);
+        this.type = type;
+        this.nric = nric;
         this.name = name;
         this.phone = phone;
-        this.email = email;
-        this.address = address;
-        this.tags.addAll(tags);
-    }
-
-    public Name getName() {
-        return name;
     }
 
-    public Phone getPhone() {
-        return phone;
+    public Type getType() {
+        return type;
     }
 
-    public Email getEmail() {
-        return email;
+    public Nric getNric() {
+        return nric;
     }
 
-    public Address getAddress() {
-        return address;
+    public Name getName() {
+        return name;
     }
 
-    /**
-     * Returns an immutable tag set, which throws {@code UnsupportedOperationException}
-     * if modification is attempted.
-     */
-    public Set<Tag> getTags() {
-        return Collections.unmodifiableSet(tags);
+    public Phone getPhone() {
+        return phone;
     }
 
     /**
-     * Returns true if both persons of the same name have at least one other identity field that is the same.
+     * Returns true if both persons of the same nric.
      * This defines a weaker notion of equality between two persons.
```

Comment:
I think this sentence is no longer relevant. NRIC uniquely identifies a person

Reply:
This one is for object comparison. So for weaker comparison, we only check the nric, but stronger one we check whether the two of them actually point to the same object. So i think still needed.